### PR TITLE
fix(dataloaders): change async request to built-in

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: ABORTED, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/626/

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: ABORTED, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/632/

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/633/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

File: pyTigerGraph/gds/dataloaders.py [hunk 1]
```diff
@@ -474,9 +466,7 @@ def _request_kafka(
             return
             
         while not exit_event.is_set():
-            status = tgraph._get(
-                tgraph.restppUrl + "/query_status", params=_stat_payload
-            )
+            status = tgraph.checkQueryStatus(resp["request_id"])
```
Comment (billshitg) [inline pyTigerGraph/gds/dataloaders.py:469]: Doesn't `runInstalledQuery` return request_id directly?

File: pyTigerGraph/gds/dataloaders.py [hunk 2]
```diff
@@ -449,17 +449,9 @@ def _request_kafka(
...
         _payload.update(payload)
-        resp = tgraph._post(
-            tgraph.restppUrl + "/query/" + tgraph.graphname + "/" + query_name,
-            data=_payload,
-            headers=_headers,
-            resKey=None
-        )
+        resp = tgraph.runInstalledQuery(query_name, params=_payload, timeout=timeout, usePost=True, runAsync=True)
         # Check status
         try:
```
Comment (billshitg) [inline pyTigerGraph/gds/dataloaders.py:456]: Seems this try-catch block is no longer needed.

File: pyTigerGraph/gds/dataloaders.py [hunk 3]
```diff
@@ -474,9 +466,7 @@ def _request_kafka(
             return
             
         while not exit_event.is_set():
-            status = tgraph._get(
-                tgraph.restppUrl + "/query_status", params=_stat_payload
-            )
+            status = tgraph.checkQueryStatus(resp["request_id"])
             if status[0]["status"] == "running":
                 sleep(1)
                 continue
```
Comment (billshitg) [inline pyTigerGraph/gds/dataloaders.py:472]: Might want to change the get function two lines below with getQueryResult as well?

File: pyTigerGraph/gds/dataloaders.py (hunk 1 above)
Comment (parkererickson-tg) [reply pyTigerGraph/gds/dataloaders.py:469]: Good catch

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/642/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Pull request merged**